pkg/distro/rhel8/distro_test.go

```diff
@@ -489,7 +489,7 @@ func TestDistro_ManifestError(t *testing.T) {
 			if imgTypeName == "edge-commit" || imgTypeName == "edge-container" {
 				assert.EqualError(t, err, "kernel boot parameter customizations are not supported for ostree types")
 			} else if imgTypeName == "edge-raw-image" {
-				assert.EqualError(t, err, "edge raw images require specifying a URL from which to retrieve the OSTree commit")
+				assert.EqualError(t, err, fmt.Sprintf("\"%s\" images require specifying a URL from which to retrieve the OSTree commit", imgTypeName))
```

Comment:
nitpick: I know that this is consistent with the message below, but we could use `%q` instead of `\"%s\"`.

Reply:
Yes, totally.  I copy pasted from RHEL 9 but I much prefer `%q` too.